### PR TITLE
INC-865 Return Default level for IEP from Prison API endpoint

### PR DESCRIPTION
**Pull request by Mjwillis**
- INC-865 Return Default level for IEP from Prison API endpoint

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/IepLevel.java [hunk 1]
```diff
@@ -26,4 +26,7 @@ public class IepLevel {
 
     @Schema(description = "Sequence Order of IEP", example = "1")
     private Integer sequence;
+
+    @Schema(description = "Indicates this is the default level for this prison", example = "false", type = "Boolean")
+    private boolean defaultLevel;
```
Comment (xoen) [inline src/main/java/uk/gov/justice/hmpps/prison/api/model/IepLevel.java:31]: Hi, would be possible to name this field `default` please 🙏 ☺️

Comment (Mjwillis) [reply src/main/java/uk/gov/justice/hmpps/prison/api/model/IepLevel.java:31]: no

Comment (xoen) [reply src/main/java/uk/gov/justice/hmpps/prison/api/model/IepLevel.java:31]: Looks like `default` is reserved word so it's not possible, ignore me :)

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**